src/database/schema.sql

```diff
@@ -49,4 +49,18 @@ CREATE TABLE books (
   publisher VARCHAR(200) NOT NULL,
   cover_state VARCHAR(4) NOT NULL,
   PRIMARY KEY(id)
+);
+
+-- Music Album class table
+CREATE TABLE music_album (
```

Comment:
Great job removing the unnecessary attributes but you still need to include columns for ALL of the properties inherited from the Item class (genre, author, label). Take a look at how the game and book tables are modeled for a clue.